tests/test_bc.py

```diff
@@ -43,17 +42,13 @@ def test_bc(trainer: bc.BC, venv):
 
 
 def test_save_reload(trainer, tmpdir):
-    pol_path = os.path.join(tmpdir, "policy.pkl")
+    pol_path = os.path.join(tmpdir, "policy.pt")
     # just to change the values a little
     trainer.train(n_epochs=1)
-    var_values = trainer.sess.run(trainer.policy_variables)
+    var_values = [param for param in trainer.policy.parameters()]
```

Comment:
Use `list(trainer.policy.parameters()` instead?

Reply:
Nice. I'll do that!